### DISCUSSION TITLE
Smart Maps for UNIFIL: Language Categories

### DISCUSSION THREAD

**Discussion by hfu** [UNopenGIS/7#425] [Ideas]
https://github.com/UNopenGIS/7/issues/254#issuecomment-1694596545 と同様なアプローチを経て、Smart Maps for UNIFIL で取り扱うべき言語について、次のようなカテゴリ分けが可能ではないかと思いました。

# カテゴリーA: 作業言語 (working languages)
国連作業言語である英語、仏語にレバノンの公用語であるアラビア語を加えた３言語です。
- 英語
- 仏語
- アラビア語（レバノンの公用語）

# カテゴリーB: 貢献国公用語 (contributor official languages)
- インドネシア語
- イタリア語
- ヒンディ語
- ガーナ語（？）→公用語は英語で、地域語が多いようです。
- ネパール語
- マレーシア語
- スペイン語
- 中国語
- アイルランド語

# カテゴリーC: 貢献国地域語 (contributor regional languages)
貢献国によっては、対応するべき地域語があるかもしれません。未確認ですが、次のような言語があるかもしれません。
- インドの地域言語（例えばベンガル語、テルグ語、タミル語）
- 中国の地域言語（例えば広東語、上海語、四川語）
- ガーナの地域言語も、ニーズによってはこの「カテゴリーC」で扱えば良いかも。

# カテゴリーD: スマート地図言語 (Smart Maps languages)
国連スマート地図グループ貢献者の言語です。開発のクオリティやモチベーションを高めるとともに、プロトタイプの性能評価や開発の効率化のためにも対応するべきところがあると思います。
- 日本語
- タイ語

@yuiseki やや作業が重複してしまって申し訳ありませんが、コメントあればいただけると嬉しいです。

**Comment by yuiseki** [comment]
@hfu 素晴らしい整理をありがとうございます！

一点だけ気になっているのが、カテゴリーBにおいて挙げられている、「ガーナ語」と呼ばれている言語は、無さそうであるということです。
Wikipediaにも「ガーナ語」という項目がそもそもありません。
「ガーナの言語」という項目はあります： https://ja.wikipedia.org/wiki/%E3%82%AC%E3%83%BC%E3%83%8A%E3%81%AE%E8%A8%80%E8%AA%9E
> ガーナは、約80の言語が使用されている[多言語](https://ja.wikipedia.org/wiki/%E5%A4%9A%E8%A8%80%E8%AA%9E)国家である。その中で、[英語](https://ja.wikipedia.org/wiki/%E8%8B%B1%E8%AA%9E)は[公用語](https://ja.wikipedia.org/wiki/%E5%85%AC%E7%94%A8%E8%AA%9E)および[リングワ・フランカ](https://ja.wikipedia.org/wiki/%E3%83%AA%E3%83%B3%E3%82%B0%E3%83%AF%E3%83%BB%E3%83%95%E3%83%A9%E3%83%B3%E3%82%AB)（通商語）として使用されている。先住民族の言語の中で最も広範囲で話されているのは[アカン語](https://ja.wikipedia.org/wiki/%E3%82%A2%E3%82%AB%E3%83%B3%E8%AA%9E)である。

以上の記述を信用するなら、ガーナでは英語が公用語であり、カテゴリーCにアカン語やトゥイ語（チュイ語？）といった地域語を含めるのが良さそうです。

**Comment by hfu** [comment]
@yuiseki ありがとうございます！ガーナについて、その通りだと思います。

リストアップした言語を、類似性で分類すると、たぶん次のようになるみたいです。

## ゲルマン諸語
- **英語**
- アイルランド語

## アラビア諸語
- **アラビア語**

## ロマンス諸語
- **仏語**
- イタリア語
- スペイン語

## 南アジア諸語
- ヒンディ語
- ネパール語

## 東南アジア諸語
- インドネシア語
- マレーシア語
- タイ語

中国語と日本語はそれぞれ、ということで良いと思います。

ロマンス諸語、南アジア諸語、東南アジア諸語の内部類似性についてある程度理解を深めておこうと思いました。

**Comment by hfu** [comment]
ロマンス諸語、南アジア諸語、東南アジア諸語の内部類似性について Chat GPT 3.5 に聞いてみたところ、概ねどれも「総合的に言って、これらの言語は類似性を持つが、それぞれ独自の言語として発展してきたため、差異もあります。互いに理解できる程度の類似性があるかもしれませんが、異なる言語として認識されています。」というくらいの関係性であるという応答でした。

**Comment by hfu** [comment]
改めて振り返ってみると、当座の言語戦略としては、①英語、②フランス語、③アラビア語をメインとし、余力があれば④ヒンディ語、⑤インドネシア語にもチャレンジしてみて、さらに諸語内部の展開を試みる、というようなことが考えられるのかなと思います。他方で、①、②はともかく、③、④、⑤については現在グループ内に話せる人がいないはずであり、我々は現在多言語化に取り組める体制になっているか、ということは再考の価値があるように思います。

今後、アラビア語が使える人の参加を得られた時点で改めて考える、というのも一案かもしれません。

他にアプローチがあるとすれば、現在メンバーが母語又は母語に近いレベルで知っている言語、例えば日本語、韓国語、タイ語といった言語で試していくというアプローチです。現在のグループから出発して参加者を増やしていくためのツールになるかもしれません。また、MUNDO的アプローチで国際案件を自分ごととして考えていくきっかけにしていく、という効果もあると思います。

いったん、本件のプライオリティを下げます。

**Comment by hfu** [comment]
#274

**Comment by takano32** [comment]
Issue として閉じられているのが惜しいと思いました。

Discussion などを活用してもよいかもしれません。

**Comment by hfu** [comment]
@takano32 こんにちは。ありがとうございます！

リオープンして Discussion に変更してみますね。
